public class tryCatchexample 
{  
  
    public static void main(String[] args) 
    {  
        try  
        {  
        int data=50/0; // throw exception   
        }  
             
        catch(ArithmeticException e)  //handling the exception 
        {  
            System.out.println(e);  
        }  
        System.out.println("remining of the code");  
    }  
      
}  